### PR TITLE
CIS SDK for filters and Firewall rules

### PR DESCRIPTION
**Pull request by DwitikrDash**
## PR summary

**Fixes:** <! -- link to issue -->

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [ ] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [X] Feature
- [X] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [X] New tests
- [ ] Build/CI related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [X] No

### CODE REVIEW

**Review by kennburger — CHANGES_REQUESTED** [review]

File: filtersv1/filters_v1_integration_test.go [hunk 1]
```diff
@@ -0,0 +1,335 @@
+/*
+ * (C) Copyright IBM Corp. 2020.
```
[the remark this answers is not in the document]
Comment (DwitikrDash) [reply filtersv1/filters_v1_integration_test.go]: Updated to 2021

File: firewallrulesv1/firewall_rules_v1_integration_test.go [hunk 2]
```diff
@@ -0,0 +1,480 @@
+/*
+ * (C) Copyright IBM Corp. 2020.
```
[the remark this answers is not in the document]
Comment (DwitikrDash) [reply firewallrulesv1/firewall_rules_v1_integration_test.go]: Updated to 2021

File: sslcertificateapiv1/ssl_certificate_api_v1_integration_test.go [hunk 3]
```diff
@@ -163,12 +163,8 @@ var _ = Describe(`sslcertificateapiv1`, func() {
 				Expect(orderResp).ToNot(BeNil())
 				Expect(orderResult).ToNot(BeNil())
 				Expect(orderResult.Result.ID).ToNot(BeNil())
-				Expect(orderResult.Result.PrimaryCertificate).ToNot(BeNil())
-				Expect(orderResult.Result.Hosts[0]).Should(BeEquivalentTo(url))
 				Expect(orderResult.Result.Status).ToNot(BeNil())
-				Expect(*orderResult.Result.Type).Should(BeEquivalentTo(OrderCertificateOptions_Type_Dedicated))
-				Expect(orderResult.Result.Certificates[0].Hosts[0]).Should(BeEquivalentTo(url))
-				Expect(orderResult.Result.Certificates[0].ID).ToNot(BeNil())
+				Expect(*orderResult.Result.Type).Should(BeEquivalentTo("advanced"))
```
[the remark this answers is not in the document]
Comment (DwitikrDash) [reply sslcertificateapiv1/ssl_certificate_api_v1_integration_test.go:167]: Updated to 2021

**Review by csulrong — APPROVED** [review]

**Review by MalarvizhiK — APPROVED** [review]
LGTM

**Review by kennburger — APPROVED** [review]

**Pull request merged**